### PR TITLE
add new bash.bashrc file to avoid segfault aarch64

### PR DESCRIPTION
**Pull request by kuburan**
almost of aarch64 architecture have segfault after upgrade, problem solved by @xeffyr [#2066](https://github.com/termux/termux-packages/issues/2066)

### CODE REVIEW

File: packages/bash/aarch64-etc-bash.bashrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+command_not_found_handle() {
+	@TERMUX_PREFIX@/libexec/termux/command-not-found "$1"
+}
+
+{
+unset LD_PRELOAD
```
Comment (tomty89) [inline packages/bash/aarch64-etc-bash.bashrc]: Why {} around it?

**Comment by tomty89** [comment]
Doesn't feel right about this anyway. It might be a problem on your phone which happened to be aarch64, but how do you know it is even limited to that?

If termux-exec has compatibility issue, perhaps it simply shouldn't be made mandatory? (i.e. pulled by apt) @fornwall

**Comment by Grimler91** [comment]
It works fine on my aarch64 device, I haven't been able to reproduce the error you (and others) are having, so it might only happen on some android versions or some roms. So, as @tomty89 said, this is probably not the right way.

**Comment by kuburan** [comment]
oh sorry, i was wrong about that.

**Pull request closed without merging**